.github/workflows/ci.yml

```
@@ -20,9 +20,12 @@ jobs:
             RabbitMQ,
             RavenDB4,
             SqlServer,
+            Oracle,
+            Oracle-RabbitMQ,
             AzureStorage-RabbitMQ,
             AzureStorage-RavenDB4,
             AzureStorage-SqlServer,
+            AzureStorage-Oracle,
             RabbitMQ-RavenDB4,
             RabbitMQ-SqlServer
```

When we created this workflow, we tried to keep these in alphabetical order, within singular/multiple groups, and the multiple groups use the `transport-peristence` convention:

```suggestion
            Oracle,
            RabbitMQ,
            RavenDB4,
            SqlServer,
            AzureStorage-Oracle,
            AzureStorage-RabbitMQ,
            AzureStorage-RavenDB4,
            AzureStorage-SqlServer,
            RabbitMQ-Oracle,
            RabbitMQ-RavenDB4,
            RabbitMQ-SqlServer
```